explorer/scripts/healthcare_gov_url_to_csv.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/local/bin/python3.6
+"""
+Create csvs from healthcare.gov data extracted from files or from the website directly.
+
+The data is filtered by plan_ids for a given state, and by provider address to include
+only in-state or neighboring state (if provided) addresses.
+
+Csvs are output into data/healthcare_gov/[issuer_id].csv
+
+This script will also output the number of providers per plan.
+"""
+import argparse
+import ijson
+import json
+import logging
+import os
+import urllib
+
+import pandas as pd
+
+from lib import etl_helper
+
+logging.basicConfig(level=logging.INFO)
+
+
+def normalize_provider(provider, plan_ids_set):
+    """Flatten provider json blobs and filter by plan id."""
+    normalized = []
+    addresses = provider.pop('addresses')
+    plans = provider.pop('plans')
+    specialties = provider.pop('specialty')  # this is a list
+    name = provider.pop('name')
+    for plan in plans:
+        if plan['plan_id'] not in plan_ids_set:
+            continue
+        for address in addresses:
+            for specialty in specialties:
+                to_append = {**address, **plan, **name, **provider}
+                to_append['specialty'] = specialty
+                normalized.append(to_append)
+    return normalized
+
+
+def filter_by_state(providers, states):
```

Comment:
So this needs to receive a list? Should it be filter_by_stateS ?

Reply:
fixed